docs/managing_deis/operational_tasks.rst

```diff
@@ -52,28 +52,22 @@ Re-issuing User Authentication Tokens
 The controller API uses a simple token-based HTTP Authentication scheme. Token authentication is
 appropriate for client-server setups, such as native desktop and mobile clients. Each user of the
 platform is issued a token the first time that they sign up on the platform. If this token is
-compromised, you'll need to manually intervene to re-issue a new authentication token for the user.
-To do this, SSH into the node running the controller and drop into a Django shell:
+compromised, it will need to be regenerated.
+
+A user can regenerate their own token like this:
 
 .. code-block:: console
 
-    $ fleetctl ssh deis-controller
-    $ docker exec -it deis-controller python manage.py shell
-    >>>
+    $ deis auth:regenerate
 
-At this point, let's re-issue an auth token for this user. Let's assume that the name for the user
-is Bob (poor Bob):
+A admin can also regenerate the token of another user like this:
```

Comment:
An*

We should alsouse the fully-formed "administrator" here, since this is formal documentation. :necktie: